filecoindot-ocw/src/result.rs

```diff
@@ -2,14 +2,22 @@
 // SPDX-License-Identifier: LGPL-3.0-only
 use derive_more::Display;
```

Comment:
use thiserror::Error instead of only deriving Display? thiserror does that anyway

Reply:
should use `Display` since `thiserror` contains `std` and could not compile into runtime

Reply:
ah yes, of course we can't use `std::error`